### ISSUE TITLE
Improve Colormap example.  

### ISSUE THREAD

**Issue by jklymak** [matplotlib/matplotlib#9531]
### Bug report
 
Just a placeholder for the discussion in #9530 including possible improvements to the colormap example. 

```python
# Paste your code here
#
#
```

**Actual outcome**

```
# If applicable, paste the console output here
#
#
```

**Expected outcome**

**Matplotlib version**

  * Operating system:
  * Matplotlib version: 
  * Matplotlib backend (`print(matplotlib.get_backend())`):
  * Python version:
  * Jupyter version (if applicable):
  * Other libraries:

**Comment by jklymak** [comment]
See also #9532

**Comment by jklymak** [comment]
Closed by #9532
